examples/oft-solana/tasks/solana/initConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { Keypair, PublicKey } from '@solana/web3.js'
+import { ConfigurableTaskDefinition } from 'hardhat/types'
+
+import { inheritTask } from '@layerzerolabs/devtools-evm-hardhat'
+import { type LogLevel } from '@layerzerolabs/io-devtools'
+import { type OAppConfigurator } from '@layerzerolabs/ua-devtools'
+import { TASK_LZ_OAPP_WIRE } from '@layerzerolabs/ua-devtools-evm-hardhat'
+import { initOFTAccounts } from '@layerzerolabs/ua-devtools-solana'
+
+// We'll create clones of the wire task and only override the configurator argument
+const wireLikeTask = inheritTask(TASK_LZ_OAPP_WIRE)
+
+/**
+ * Additional CLI arguments for our custom wire task
+ */
+interface Args {
```

Comment:
nit: Seems like this interface keeps getting declared in multiple files. Can we just move it to a shared utils file or something?

Reply:
actually just realized this is pretty different for most files but happened to be the same for [oft-solana/tasks/common/wire.ts](https://github.com/LayerZero-Labs/devtools/pull/1268/files/51282b39b2c5bb6240840f4ea2ab7245fac7edc4#diff-b461216ddbe5374dd56ed42b2607cb53e9dabc6f6fe0252a879f1c95c6aa2dd9L25-L31)

feel free to ignore in this case